infrastructure/zk/src/init.ts

```diff
@@ -138,6 +133,67 @@ export async function submoduleUpdate() {
     await utils.exec('git submodule update');
 }
 
+async function configMode(validiumMode: boolean) {
```

Comment:
We can abstract the logic that loads and updates both `chain.toml` and `eth_sender.toml` files into different functions (`updateChainConfig`, and `updateEthSenderConfig`) to improve the code's readability.